src/dhali/module.py

```diff
@@ -0,0 +1,34 @@
+import json
+import requests
+from xrpl import wallet
+
+class Module:
+    def __init__(self, asset_uuid: str, xrpl_wallet: wallet):
+        self.asset_uuid = asset_uuid
+        self.xrpl_wallet = xrpl_wallet
+        self.url = f"https://dhali-prod-run-dauenf0n.uc.gateway.dev/{asset_uuid}/run"
+
+    def run(self, input_file_path: str, payment_claim: dict,):
+        headers = {'Payment-Claim': json.dumps(payment_claim)}
+        
+        with open(input_file_path, 'rb') as f:
+            files = {'input': f}
+            response = requests.put(self.url, headers=headers, files=files)
+
+        return response
```

Comment:
Rather than passing a path to a file, would it be better to path a file-like object here?

Reply:
Will update